util/mbed/mbed-copy-osx.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
I agree. As you see, this script didn't make it very far...I ran into issues with the whole xml format coming from the `system_profiler` command.
I also discovered [mbed-ls](https://github.com/ARMmbed/mbed-ls) during this adventure. They claim it's cross platform, but it doesn't work for me on OSX. Works great in Ubuntu however.